layouts/solo/main-a.html

```diff
@@ -31,7 +31,7 @@
   var s, as = d.getElementsByTagName(t)[0];
   if (d.getElementById(id)) return;
   s = d.createElement(t); s.id = id;
-  s.src = '//d2kmhr1caomykv.cloudfront.net/player/assets/tvp/tvp-{{ ($.Scratch.Get "settings").version }}-min.js';
+  s.src = "http://localhost:1313/solo/player-debug.js";
```

Comment:
Yes, Ricky is going to fix the player lib and make a new version.
Until that we put the fix into a player-debug.js